prov/rxm/src/rxm_cq.c

```diff
@@ -1009,6 +1003,34 @@ static inline ssize_t rxm_ep_read_msg_cq(struct rxm_ep *rxm_ep)
 	return ret;
 }
 
+static inline ssize_t
+rxm_conn_handle_cmap_cmd_queue(struct rxm_ep *rxm_ep)
+{
+	ssize_t ret = 0;
+	struct util_cmap_cmd *cmd;
+
+	while (!dlist_ts_empty(&rxm_ep->util_ep.cmap->cmd_queue)) {
+		dlist_ts_pop_front(&rxm_ep->util_ep.cmap->cmd_queue,
+				   struct util_cmap_cmd, cmd, entry);
+		if (!cmd)
+			continue;
+		rxm_ep->util_ep.cmap->cmd_read++;
+		switch (cmd->type) {
+		case UTIL_CMAP_CMD_AV_UPD:
+			free(cmd);
+			rxm_ep->util_ep.cmap->av_upd_cmd = NULL;
```

Comment:
We need to be holding cmap->lock when updating this variable. This can probably be pushed inside reprocess recv queue function where we acquire the lock.